index.html

```diff
@@ -24,11 +24,26 @@
                     </div>
                 </nav>
             </div>
+
+            <div class="desktop-header">
+              <div class="logo">
+                <a href=""><img src="My Logo.png" alt="" class="logo-img" /></a> 
+             </div>
+             <nav>
```

Comment:
- [ ] Good working on some of the changes requested but there are some aspects you can still improve on.
  - Your desktop `nav bar` is still visible on the mobile version. Kindly remove it as it is only supposed to be visible in the desktop version
  
![image](https://github.com/Developer-Kingz/my-portforlio-project/assets/98453979/dbe8e608-f4ca-4c9d-8491-41fc8d259f20)

 - As requested earlier, Kindly improve the spacing between the elements to match the original design
 
![image](https://github.com/Developer-Kingz/my-portforlio-project/assets/98453979/79551abc-4dda-44a5-a1d9-934a6c4b98f1)

![image](https://github.com/Developer-Kingz/my-portforlio-project/assets/98453979/14f39fea-d5f1-47cd-be94-d2efc3d005d2)

 - Check out the about me section, the social icons appear to be out of position together with the `get my resume` button. Kindly position them correctly
![image](https://github.com/Developer-Kingz/my-portforlio-project/assets/98453979/6d15ee4a-5dbb-4a6c-a2c6-dd76d7d1fd74)